skimage/data/_registry.py

```diff
@@ -25,6 +25,8 @@
     'logo.png',
     'microaneurysms.png',
     'moon.png',
+    'multipage_rgb.tif',
+    'no_time_for_that_tiny.gif',
```

Comment:
It looks like `legacy_datasets` is only ever used to create `legacy_registry`. And it seems the latter is never used. So this change might accomplish nothing?

Tagging @hmaarrfk in case you have more insights.

Reply:
Hmm, might actually been me who touched this most recently in https://github.com/scikit-image/scikit-image/pull/6886.

Reply:
This is the problem we need to address: https://github.com/scikit-image/scikit-image/issues/6993#issuecomment-2585859683

Reply:
I created an issue for this in order to not hold this up further: https://github.com/scikit-image/scikit-image/issues/7676

Their should be no harm in keeping this variable around for now.